### ISSUE TITLE
Potential Missing Filtering of -1 Labels in rcnn_cls_labels During Binary Cross-Entropy Loss Calculation in PointRCNN

### ISSUE THREAD

**Issue by ChunhengH** [open-mmlab/OpenLidarPerceptron#1682]
I was training pointRCNN and other point-based models using my own dataset and noticed that rcnn_cls_labels came up -1, which caused the training to terminate. Comparing to the original model I found that it seems that the -1 is not filtered when doing the binary cross entropy loss calculation. pointRCNN original model uses cls_valid_mask to filter all labels greater than 0, but in pcdet/models/roi_heads/roi_head_template.py it seems that not perform this step. I'm not sure if this is a problem.

**Comment by hdddhdd** [comment]
I am also experiencing the same error.. have you found a solution yet?
